marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/AllArtifactsProject.java

```diff
@@ -34,20 +34,20 @@ public AllArtifactsProject(HubClient hubClient) {
      * Writes the zip to a file and reads all the entries into memory.
      */
     public void writeProjectArtifactsToZipFile() {
-        try {
-            zipFile = new File("build/allArtifactsProject.zip");
-            FileOutputStream fos = new FileOutputStream(zipFile);
+        try{
```

Comment:
And "writeProjectUpdatesToZipFile", and so on for other uses of the term "project artifacts" in this class.